app/tab_input_data.py

```diff
@@ -33,15 +33,10 @@ def content_input_data(api: PtxboaAPI) -> None:
             horizontal=True,
         )
 
-        # in order to keep the figures horizontally aligned, we create two st.columns
-        # pairs the columns are identified by c_{row}_{column}, zero indexed
-        c_0_0, c_0_1 = st.columns([2, 1], gap="large")
-        c_1_0, c_1_1 = st.columns([2, 1], gap="large")
-        with c_0_0:
-            st.markdown("**Map**")
+        with st.expander("**Map**"):

```

Comment:
I would prefer to see the map by default:
```suggestion
        with st.expander("**Map**", expanded=True):

```